bin/pt-pmp

```diff
@@ -671,22 +708,57 @@ main() {
             OPT_PID=$(ps -eaf | grep "$OPT_BINARY" | grep -v grep | awk '{print $2}' | head -n1);
          fi
       fi
+      
+      DUMPER=""
+      if [ $OPT_DUMPER == "gdb" ]; then
+      	 if [ `_which gdb` ]; then
+      	 
+      	    readnever=""
+      	    if [ $OPT_READNEVER ]; then
+               if gdb -nx --quiet --batch --readnever > /dev/null 2>&1; then
+                  readnever="--readnever"
+               else
+                  die "gdb does not support option --readnever. Exiting"
+               fi
+            fi
+         
+            DUMPER="
+               gdb ${readnever} -ex 'set pagination 0' \
+                   -ex 'thread apply all bt' \
+                   -batch                    \
+                   -p ${OPT_PID}
+               "
+         else
+         	die "Can't find gdb binary. Exiting"
+         fi
+      elif [ $OPT_DUMPER == "eu" ]; then
+      	 if [ `_which eu-stack` ]; then
+      	    DUMPER="eu-stack -s -m -p ${OPT_PID} | sed -e '\$!N;s/\n //g;P;D' | sed -e 's/\(0x[[:xdigit:]]*\) -/\1 ??() -/'"
+      	 else
+      	 	die "Can't find eu-stack binary from elfutils. Exiting"
+      	 fi
+      elif [ $OPT_DUMPER == "pteu" ]; then
+      	 if [ `_which eu-stack` ] && [ `_which pt-eustack-resolver` ]; then
+      	    DUMPER="pt-eustack-resolver ${OPT_PID}"
+      	 else
+      	 	die "Can't find eu-stack binary from elfutils or pt-eustack-resolver. Exiting"
```

Comment:
Message is confusing. It's better to generate separate messages for eu-stack and for pt-eustack-resolver.
E.g. 
die "Can't find eu-stack binary from elfutils package"
and
die "Can't find pt-eustack-resolver tool in PATH."

If the percona toolkit is installed from tar.gz, pt-eustack-resolver could be not in $PATH, but in the same directory as pt-pmp

Reply:
Updated, please check.